app/src/main/java/io/appium/uiautomator2/utils/Device.java

```diff
@@ -22,6 +25,14 @@
 public abstract class Device {
 
     public static UiDevice getUiDevice() {
+        boolean disableSuppressAccessibilityService = Boolean.parseBoolean(
+                InstrumentationRegistry.getArguments().getString("DISABLE_SUPPRESS_ACCESSIBILITY_SERVICES", "false"));
+        if (disableSuppressAccessibilityService && Build.VERSION.SDK_INT >= Build.VERSION_CODES.N) {
+            // The flag is necessary not to stop running accessibility service
+            // https://github.com/appium/appium/issues/4910
+            // https://developer.android.com/reference/android/app/UiAutomation.html#FLAG_DONT_SUPPRESS_ACCESSIBILITY_SERVICES
+            Configurator.getInstance().setUiAutomationFlags(UiAutomation.FLAG_DONT_SUPPRESS_ACCESSIBILITY_SERVICES);
```

Comment:
should we do this each time we get the instrumentation or it would be enough to call the configurator once?